chapters/revisions.tex

```diff
@@ -1265,7 +1265,7 @@ \subsection{Contributors to the Modelica Language, Version 3.2}\label{contributo
 and demonstrated its use on a vehicle dynamics example. Utilizing the
 prototype implementation, Francesco Casella demonstrated with a model of
 a thermal power plant with 390 iteration variables of the initialization
-problem, that an appropriate usage of the homotopy operator allows to
+problem, that an appropriate usage of the homotopy operator lets one
```

Comment:
I don't recognize us using _one_ in this way.  If that's correct, maybe we shouldn't start doing it now?  For instance, why not this?
> problem, that an appropriate usage of the homotopy operator allows the system to be initialized reliably without…

Reply:
Yes, that makes sense - I have changed that now. It was only done in the revisions chapter.